addons/mail/tests/test_mail_template.py

```diff
@@ -147,6 +147,12 @@ def test_server_archived_usage_protection(self):
         server.action_archive()  # No more usage -> can be archived
         self.assertFalse(server.active)
 
+    def test_template_translation_invalid_language(self):
```

Comment:
This test should be done in base as it has nothing to do with mail itself ... we already have a lot of tests, no need to add tests related to the base translation mechanism.

Reply:
Agree: we should test invalid languages in the `base`
But we should test the
```
for lang in translations:
    self.with_context(lang=lang)._check_access_right_dynamic_template()
```
logic here.
Try to update a translation with content which will raise Error in `_is_dynamic`

Reply:
We have another task for that, will be done soon with other mail tests, as it is a bit secundary :)